script/reset-environment.sh

```diff
@@ -1,13 +1,20 @@
 #!/usr/bin/env bash
+if ! [ -x "$(command -v yarn)" ]; then
+    echo "Installing yarn..."
+    npm i -g yarn@0.21.3
+    if [ $? -eq 0 ]; then
+        echo "Yarn successfulling installed globally."
+    fi
```

Comment:
Maybe add one here if yarn fails to be installed for some reason? Just because you have such nice else conditions below. 😉